src/sentry/integrations/mixins/issues.py

```diff
@@ -8,15 +8,14 @@
 from sentry.integrations.utils import where_should_sync
 from sentry.models import ExternalIssue, GroupLink, User, UserOption
 from sentry.notifications.utils import (
+    get_group_title,
```

Comment:
can we change this to `get_group_title as get_notification_group_title` or something to prevent the duplicate naming?